templates/tutorialv2/view/content.html

```diff
@@ -230,6 +235,9 @@ <h2>
             <h2>
                 <a class="force-blue" href="{% url "content:create-extract" content.pk content.slug %}">
                     {% trans "Ajouter une section" %}
+                </a> {% trans 'ou'%}
+                <a class="force-blue" href="{% url "content:create-quizz" content.pk content.slug %}">
+                    {% trans "un quizz" %}
```

Comment:
Pour la sémantique du lien
```suggestion
                    {% trans "ajouter un quizz" %}
```